api/builtins/HelmChartInflationGenerator.go

```diff
@@ -267,6 +267,9 @@ func (p *HelmChartInflationGeneratorPlugin) templateCommand() []string {
 	if p.ReleaseName != "" {
 		args = append(args, p.ReleaseName)
 	}
+	if p.Namespace != "" {
```

Comment:
I'm not exactly sure where this came from. From what I can tell, this is generated by `//go:generate pluginator`. Maybe a previous commit was merged where it wasn't ran?

Reply:
that's right.